### PR TITLE
#4638 Locale size to native size

### PR DESCRIPTION
**Pull request by NFSS10**
| - | - |
| --- | --- |
| Issue | https://github.com/ripe-tech/ripe-core/issues/4638 |
| Dependencies | -- |
| Decisions | • Add `localeToNative`, `localeToNativeP`, `localeToNativeB ` and `localeToNativeBP` methodsto convert locale sizes to native sizes<br>• Add `#localeToNative()` and `#localeToNativeB()` tests  |
| Tests | imagem |

### CODE REVIEW

**Comment by NFSS10** [comment]
@BeeMargarida I'm aware the tests are failling in the GitHub checks, but that's because it's not using ripe-core-now. Check this PR description section there is proof there that the tests are working properly :)

**Review by BeeMargarida — COMMENTED** [review]
Tested locally, all tests passed :+1:

File: src/js/api/size.js [hunk 1]
```diff
@@ -350,3 +350,78 @@ ripe.Ripe.prototype.nativeToLocaleBP = function(scales, values, genders, options
         });
     });
 };
+
+/**
+ * Converts a locake size value in the specified scale to the corresponding native size.
+ * The available scales, genders and sizes can be obtained with the method getSizes.
```
[the remark this answers is not in the document]
Comment (NFSS10) [reply src/js/api/size.js]: getSizes seems to be correct according to the rest of the sizes documentation 🤷‍♂️

[the remark this answers is not in the document]
Comment (BeeMargarida) [reply src/js/api/size.js]: @NFSS10 Actually it should be by using ``, like in https://github.com/ripe-tech/ripe-sdk/blob/e51f9fb21635a84aa53b01ecc6eaaa22cc8b6e06/src/js/base/ripe.js#L792

[the remark this answers is not in the document]
Comment (NFSS10) [reply src/js/api/size.js]: Okay, we should update the rest of the docs then

**Review by BeeMargarida — APPROVED** [review]

**Pull request merged**